cartographer/mapping/internal/3d/pose_graph/optimization_problem_3d_test.cc

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-#include "cartographer/mapping/3d/pose_graph/optimization_problem_3d.h"
+#include "optimization_problem_3d.h"
```

Comment:
I haven't seen this for other tests (just make sure it's consistent)

Reply:
ah, that's what CLion does sometimes. Thanks for noticing, I thought I cleaned up all of these.